### PR TITLE
docs(python): add assertion examples

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/intro-java.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ public class Example {
     <dependency>
       <groupId>com.microsoft.playwright</groupId>
       <artifactId>playwright</artifactId>
-      <version>1.16.0</version>
+      <version>1.17.0</version>
```
Comment (pavelfeldman) [inline docs/src/intro-java.md:62]: Looks unrelated

Comment (mxschmitt) [reply docs/src/intro-java.md:62]: it is a drive-by.

Comment (pavelfeldman) [reply docs/src/intro-java.md:62]: Yes, but won't we be merging this into docs for 1.17?

Comment (mxschmitt) [reply docs/src/intro-java.md:62]: I don't think so that we will merge ToT docs with 1.17 docs. It would not work, since the ToT docs contain already too many unrelated changes.

Comment (pavelfeldman) [reply docs/src/intro-java.md:62]: I was not suggesting to merge ToT docs into 1.17 docs. I was suggesting to keep this separate to remember that we need to merge it into 1.17.

Comment (mxschmitt) [reply docs/src/intro-java.md]: extracted into https://github.com/microsoft/playwright/pull/10609

**Review by pavelfeldman — APPROVED** [review]
